geode-core/src/main/java/org/apache/geode/security/AuthInitialize.java

```diff
@@ -54,8 +54,8 @@ default void init(LogWriter systemLogger, LogWriter securityLogger)
   /**
    *
    * @since Geode 1.0.
-   * @deprecated in Geode 1.5, never called by the product, use init(LogWriter systemLogger,
-   *             LogWriter securityLogger)
+   * @deprecated in Geode 1.5. Never called by the product. Use {link #init(LogWriter systemLogger,
```

Comment:
Forgot the `@` in `@link`.

If you're using IntelliJ, I think the default "view javadoc" hotkey is Ctrl-J, if you want to make sure linkage is working correctly.